cloudbuild.yaml

```diff
@@ -14,7 +14,6 @@ steps:
       - kueueviz-image-push
     env:
       - IMAGE_REGISTRY=us-central1-docker.pkg.dev/k8s-staging-images/kueue
-      - IMAGE_EXTRA_TAG=$_PULL_BASE_REF
```

Comment:
Thanks 👍 I think this really makes the build simpler. I love to see the tagging strategy aligned for all images we have.